### DISCUSSION TITLE
配置文件報錯

### DISCUSSION THREAD

**Discussion by Asakurakaren** [mikumifa/biliTickerBuy#153] [提问专区]
輸出信息處，在點擊生成配置後顯示錯誤
![屏幕截图(1377)](https://github.com/mikumifa/biliTickerBuy/assets/173535315/383ac372-15c6-4f3f-9826-f4d260f920e1)

**Comment by mikumifa** [comment]
地址没填，看一下说明书

**Reply by mikumifa** [reply]
https://github.com/mikumifa/biliTickerBuy/wiki/%E6%8A%A2%E7%A5%A8%E8%AF%B4%E6%98%8E

**Reply by Asakurakaren** [reply]
thx

**Reply by Asakurakaren** [reply]
大佬請問一下bw這種還沒開票預售的展出，配置文件那裡的“選票”什麼時候會有選項，是在開始開票倒計時時候會有選項嗎

**Reply by WittF** [reply]
> 大佬請問一下bw這種還沒開票預售的展出，配置文件那裡的“選票”什麼時候會有選項，是在開始開票倒計時時候會有選項嗎

没选票选项的就是没上架票。现在[BW](https://show.bilibili.com/platform/detail.html?id=85939)就是个空项目，没有票种，只能等官方上架。开票倒计时和票上架不一定有直接关系。
